lua/dylib.lua

```diff
@@ -0,0 +1,10 @@
+-- TODO Add support for other operations. For example:
+-- Functions to add/del/list the GK FIB entries.
+-- Functions to list the ARP table.
+-- Functions to list the ND table.
+-- Functions to process the GT policies.
+-- ......
+
+function dy_default_proc()
+	return "A default message from the dynamic configuration block!"
+end
```

Comment:
Keep only the comment and drop the function `dy_default_proc()`. `dy_default_proc()` is not adding anything useful to the library.